core/src/main/java/com/amplifyframework/predictions/models/Label.java

```diff
@@ -34,10 +35,12 @@
     public static final String FEATURE_TYPE = Label.class.getSimpleName();
 
     private final List<String> parentLabels;
+    private final List<RectF> boxes;
```

Comment:
Should probably add this new field into the `hashCode()`, `toString()` and `equals()`, as well.

Reply:
this is something that needs to be applied to every result model in predictions category... will revisit this in a separate PR.